packages/authhero/test/routes/management-api/tenant-operations.spec.ts

```
@@ -0,0 +1,269 @@
+
+    expect(response.status).toBe(200);
+    expect(await response.json()).toEqual({
+      operations: [],
+      start: 0,
+      limit: 50,
+      length: 0,
+    });
+  });
+
+  it("enqueues an upgrade and exposes it via GET /operations/{id}", async () => {
+    let dataRef: DataAdapters | undefined;
```

ESLint

[error] 'dataRef' is never reassigned. Use 'const' instead.

(prefer-const)